src/components/Stepper.theme.ts

```diff
@@ -1,4 +1,4 @@
-import { keyframes } from '@chakra-ui/react'
+import { keyframes } from '@emotion/react'
```

Comment:
Not exposed from chakra after the bump, solution is to consume directly from source (this is the exact same guy)